Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/model/DocumentMetadataResult.java

```diff
@@ -25,6 +25,7 @@
  */
 package gov.hhs.fha.nhinc.model;
 
+import java.util.Date;
```

Comment:
Is this the datamodel class for our current DQ adapter?  Doesn't the hibernate mapping and dbscript need to be updated too then?